<launch>

  <!-- <arg name="pointCloud_topic_" default="/velodyne_points_ma" />
  <arg name="frame_id_" default="base_link" />
  <arg name="order" default="3" />
  <arg name="minClusterSize" default="100" />
  <arg name="clusterRadius" default="1.0" />
  <arg name="passX_min" default="-5.0" />
  <arg name="passZ_min" default="-0.8" />
  <arg name="passZ_max" default="0.0" /> -->

  <arg name="pointCloud_topic_" default="/simuSegSave/cloud_Combined" />
  <arg name="frame_id_" default="vehicle_base_link" />
  <arg name="order" default="3" />
  <arg name="minClusterSize" default="100" />
  <arg name="clusterRadius" default="1.0" />
  <arg name="passX_min" default="-5.0" />
  <arg name="passZ_min" default="-0.8" />
  <arg name="passZ_max" default="1.0" />

  <node pkg="perception" type="laneDetection" name="laneDetection" output="screen">
    <param name="pointCloud_topic_" value="$(arg pointCloud_topic_)" />
    <param name="frame_id_" value="$(arg frame_id_)" />
    <param name="order" value="$(arg order)" />
    <param name="minClusterSize" value="$(arg minClusterSize)" />
    <param name="clusterRadius" value="$(arg clusterRadius)" />
    <param name="passX_min" value="$(arg passX_min)" />
    <param name="passZ_min" value="$(arg passZ_min)" />
    <param name="passZ_max" value="$(arg passZ_max)" />
  </node>

  <!--- TF -->
  <!-- <node pkg="tf" type="static_transform_publisher" name="velodyne2base_link" args="0 0 0 0 0 0 /velodyne /base_link 100" /> -->

  <!-- - rviz -->
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find perception)/launch/laneFit.rviz"/> -->

  <!-- - rqt_reconfigure -->
  <!-- <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure"></node> -->

  <!-- - rqt_plot -->
  <!-- <node name="rqt_plot" pkg="rqt_plot" type="rqt_plot"></node> -->

  <!-- - rqt_logger_level -->
  <!-- <node name="rqt_logger_level" pkg="rqt_logger_level" type="rqt_logger_level"></node> -->

</launch>
